impl/src/main/java/com/okta/sdk/impl/resource/DefaultOIDCApplicationBuilder.java

```diff
@@ -164,9 +164,6 @@ private Application build(Client client){
         application.setAccessibility(client.instantiate(ApplicationAccessibility.class));
         ApplicationAccessibility applicationAccessibility = application.getAccessibility();
 
-        if (Strings.hasText(loginRedirectUrl))
```

Comment:
Fixes:

`com.okta.sdk.resource.ResourceException: HTTP 400, Okta E0000001 (Api validation failed: loginRedirectUrl - The application oidc_client cannot be created/updated because the loginRedirectUrl field has been deprecated.), ErrorId oaemEljCny2SQugFaC-B5p8ew`